services/121-service/src/registration/registrations.service.ts

```diff
@@ -424,6 +428,11 @@ export class RegistrationsService {
       RegistrationStatusEnum.registered,
     );
     this.inclusionScoreService.calculateInclusionScore(referenceId);
+    this.inclusionScoreService.calculatePaymentAmountMultiplier(
```

Comment:
@jannisvisser I think this one is not needed. I commented this one out locally and it still worked. Are there scenarios where you need this?

Reply:
@RubenGeo You mean because at registration, this is already done via the setAttribute method?